validation-test/stdlib/Collection/Inputs/Template.swift.gyb

```diff
@@ -54,19 +54,15 @@ def test_methods(test):
 }%
 
 % for test in all_tests():
-// Start of file: ${test.name}
+// BEGIN ${test.name}
```

Comment:
Good point!  Feel free to remove `utils/split-generated-tests` then or merge it into `utils/split_file.py`.